pkg/pillar/cmd/client/client.go

```diff
@@ -39,6 +39,8 @@ const (
 	errorTime   = 3 * time.Minute
 	warningTime = 40 * time.Second
 	return400   = false
+	certsRetry  = 3 // try fetch certs 3 times
```

Comment:
I think you should try forever the same way that we try a selfRegister forever.

Reply:
ok, i was trying to do dynamic detection. but yes, if we are doing statically V2, try forever should be the case